app/screens/auth/set_up_screen.tsx

```diff
@@ -25,11 +26,21 @@ export default function SetUsername({ navigation }: any) {
       quality: 1,
     });
     if (!result.canceled) {
-      console.log("Couldn't load image");
       setImage(result.assets[0].uri);
     }
   };
 
+  const upload = async () => {
+    if (username === "") {
+      alert("Please enter a username.");
+    } else {
+      await firestoreCtrl.setName(getUID(), username);
+      if (image) {
+        await firestoreCtrl.setProfilePicture(getUID(), image);
```

Comment:
getUID() is used multiple times and hence it would be better to store it in a const at the beginning of the file.

Reply:
It is in a way already the case as I import the function which is a const from _@/types/Auth_.